client/src/pages/Payment/Payment.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PaymentForm from '../../components/Payment/PaymentForm';
+import StripeForm from '../../components/Payment/StripeForm';
+import { Elements } from '@stripe/react-stripe-js';
+import { loadStripe } from '@stripe/stripe-js';
+import { Modal } from '@material-ui/core';
+const stripePromise = loadStripe(process.env.STRIPE_PUBLIC_KEY ? process.env.STRIPE_PUBLIC_KEY : '');
```

Comment:
You could shorten your code like this:
`loadStripe(process.env.STRIPE_PUBLIC_KEY || '' );`